### ISSUE TITLE
make the action fail when it cannot backport

### ISSUE THREAD

**Issue by 3nids** [Gaurav0/backport#36]
having a success if the bot could not perform the backport is misleading to me.

**Comment by Gaurav0** [comment]
I agree. The problem with the original behavior (commenting on the closed pull request) is that it is easy to miss in a large repo. @runspired indicated to me he would prefer a nonworking pr to such a comment. But I'm open to better ideas.

**Comment by 3nids** [comment]
Would you be willing to to do it?

**Comment by Gaurav0** [comment]
Yes, the question is about what I should do that might be acceptable to all users.

**Comment by 3nids** [comment]
Ah reading again, I get your point.
So there are 3 outputs of the bot:

1. The exit status: currently it's always green
2. The comment in the original PR
3. The created PR.

When it's failing, there is only 2 created. I would just need 1 or 3 at least, I don't really mind which.
